Makefile

```diff
@@ -1 +1,9 @@
+SHELL := /bin/bash
+export PYTHONPATH := $(PYTHONPATH):./lambda/src
```

Comment:
Curious if this one came from another example. PYTHONPATH isn't necessarily set, and this would resolve bit oddly if not... Might need something like:

```
export PYTHONPATH := $(if $(PYTHONPATH),$(PYTHONPATH):./lambda/src,./lambda/src)
```